install/kubernetes/helm/Istio/values.yaml

```diff
@@ -0,0 +1,13 @@
+Istio:
+   auth_enabled: false 
+   initializer_enabled: false
+   mixer_enabled: true
```

Comment:
mixer should always be enabled.

Reply:
why?

Reply:
Per previous comment, might be useful to follow the config map bools. Telemetry (choose statsd or Prometheus) enabled ? Policy check enabled (implies telemetry)? Tracing enabled? 
mTLS enabled ? These are what a user cares about. Component names may be unfamiliar to end users.

Reply:
I would like to merge base first then more folks who knows helm can contribute with these, or  if no takers I will do it in the following up pr.